sphinx/source/Tutorials/Virtual Terminal Basics.rst

```diff
@@ -456,6 +491,63 @@ Here's the final code for this example:
 		return 0;
 	}
 
+Writing up the CMake
+^^^^^^^^^^^^^^^^^^^^^
+
+The CMake for this program is a bit more complex than the other examples.
+
+We'll start off like we did in "ISOBUS Hello World".
+
+.. code-block:: text
+
+	cmake_minimum_required(VERSION 3.16)
+
+	project(
+	  isobus_vt_tutorial
+	  VERSION 1.0
+	  LANGUAGES CXX
+	  DESCRIPTION "An example VT client program"
+	)
+
+	set(THREADS_PREFER_PTHREAD_FLAG ON)
+	find_package(Threads REQUIRED)
+
+	add_subdirectory("ISO11783-CAN-Stack")
+
+	add_executable(vt_example main.cpp)
+
+Looking "ISOBUS Hello World" we had this next:
+
+.. code-block:: c++
+
+	target_link_libraries(isobus_hello_world PRIVATE isobus::Isobus isobus::HardwareIntegration Threads::Threads)
+
+But like we mentioned earlier, we're now using a function (the IOP file reader) in the isobus utility library called "SystemTiming", so we need to link that too:
+
+.. code-block:: c++
+
+	target_link_libraries(isobus_hello_world PRIVATE isobus::Isobus isobus::HardwareIntegration isobus::SystemTiming Threads::Threads)
+
+We also want to move our IOP file to be in the same folder as the executable after it's built, so that it can locate it.
+We can do that with this little handy bit of CMake:
+
+.. code-block:: text
+
+	add_custom_command(
+		TARGET vt_example
+		POST_BUILD
+		COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_SOURCE_DIR}/VT3TestPool.iop
+		${CMAKE_CURRENT_BINARY_DIR}/VT3TestPool.iop)
```

Comment:
Building for Windows generates an executable in `./build/debug/VT3ExampleTarget.exe` while the object pool is moved to `./build/VT3TestPool.iop`. What we could use instead to correct this:
```suggestion
		$<TARGET_FILE_DIR:vt_example>/VT3TestPool.iop)
```
From: https://stackoverflow.com/a/56231218